### PR TITLE
new-deps

### PR DESCRIPTION
**Pull request by AndreaCensi**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,7 +11,10 @@ six
...
 future
-duckietown-challenges-daffy>=5.1.4
 zeroconf
-dt_authentication
-dt_data_api
\ No newline at end of file
+
+duckietown-challenges-daffy>=6.0.4
+dt_authentication # FIXME: add daffy
+dt_data_api # FIXME: add daffy
```
Comment (afdaniele) [inline requirements.txt:18]: These two are distro agnostic.

Comment (AndreaCensi) [reply requirements.txt:18]: really committing to make it compatible with daffy, ente, and all that for ever?

Comment (afdaniele) [reply requirements.txt:18]: yes

Comment (afdaniele) [reply requirements.txt:18]: actually, no, changed my mind

Comment (afdaniele) [reply requirements.txt:18]: let's use `dt-daffy-XYZ` though, similar to what ROS does wth `ros-kinetic-...`, `ros-noetic-...` etc.

**Pull request closed without merging**